openapi/components/schemas/GatewayAccountConfig/Aircash.yaml

```
@@ -0,0 +1,31 @@
+description: Aircash gateway configuration.
+allOf:
+  - $ref: ../GatewayAccount.yaml
+  - type: object
+    required:
+      - credentials
+    properties:
+      credentials:
+        type: object
+        description: Aircash credentials object.
+        properties:
+          privateKey:
+            writeOnly: true
+            type: string
+            description: Aircash partner private key.
+          privateKeyPass:
+            writeOnly: true
+            type: string
+            description: Aircash partner private key passphrase.
```

nitpick: 3 nouns in a row
```suggestion
            description: Private key passphrase of the Aircash partner.
```